### ISSUE TITLE
Can't get both full-text and regex search working on same service

### ISSUE THREAD

**Issue by plhosk** [arve0/feathers-mongodb-fuzzy-search#3]
I have the following service hook which matches the example in Readme:

```
app.service('/api/items').hooks({
  before: {
    all: [
      auth.hooks.authenticate('jwt'),
      search(),
      search({
        fields: ['title', 'subtitle', 'contributors', 'series', 'categories'],
        fieldsNotEscaped: ['title', 'subtitle', 'contributors', 'series', 'categories'],
      }),
    ],
  },
})
```
However, when I attempt to use a full-text search with ```find({ query: { $search: 'string' } })``` i get the error: ```Error in 'api/items' service method 'find BadRequest: You are not allowed to perform $search on field $text```

If I remove the second search({... }) from the hooks (the one that declares the regex fields) the full-text search begins to work fine.

Does this mean that on a single service only one can be used at a time (full-text or regex) but not both?

I am using the  ```feathers-mongoose``` adapter along with this package.

**Comment by arve0** [comment]
Hi! Thanks for reporting!

> Does this mean that on a single service only one can be used at a time (full-text or regex) but not both?

Yes, that is correct as of now. But I'm sure that can be fixed.

**Comment by arve0** [comment]
I'm looking into this. I believe it to be a bug in `transformSearchFieldsInQuery` https://github.com/arve0/feathers-mongodb-fuzzy-search/blob/master/index.js#L25

**Comment by arve0** [comment]
This has been fixed in `v1.1.1`.
